oc-chef-pedant/spec/api/oc_id_spec.rb

```diff
@@ -2,6 +2,38 @@
 require 'openssl'
 
```

Comment:
Would it make sense to include a comment specifying endpoints that exist but are not covered, with a reference to the unit coverage? The possible down side is remembering to update that when it changes - but Id endpoints are relatively static.